newIDE/app/src/UI/SelectField.js

```diff
@@ -29,6 +29,7 @@ type Props = {|
     flex?: 1,
     width?: 'auto',
   },
+  inputStyle?: {| fontSize: '0.875em' |},
```

Comment:
Why was this change needed?

Reply:
There is no way in the `SelectField` component to change the input font size, as done in the `TextField` component [here](https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/UI/TextField.js#L99-L103). And `0.875em` corresponds to the `body2` MUI variant

Reply:
I'll change for 14 so that it's consistent with `TextField`